src/modules/cpu/kernel/resize.hpp

```diff
@@ -222,7 +222,7 @@ RppStatus resize_nn_u8_u8_host_tensor(Rpp8u *srcPtr,
                     {
                         __m128i pRow;
                         rpp_simd_load(rpp_resize_nn_load_u8pln1, srcPtrTempChn, srcLocationColumnArray, pRow);
-                        rpp_simd_store(rpp_store4_u8_to_u8, dstPtrTempChn, pRow);
```

Comment:
Is this the only file that uses rpp_store4_u8_to_u8 function?

Reply:
@snehaa8 
Please add your observation here

Reply:
@HazarathKumarM Can you please re-check this change
I dont see this change in resize.hpp in my remap PR.

[Resize.hpp](https://github.com/r-abishek/rpp/blob/58c6b1b881a6a6fadbf2641890670429e6b98287/src/modules/cpu/kernel/resize.hpp#L225C40-L225C59)